lasy/profiles/spacetime_profile.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+
+from .profile import Profile
+
+
+class SpaceTimeProfile(Profile):

```

Comment:
Maybe this could actually be part of the `GaussianProfile` laser. See other comments.

Reply:
Yes, agreed. If we decide to have it applied with GaussianProfile, we completely change the implementation of the `GaussianProfile` (which is definitely possible): instead of calling the temporal and spatial profile separately, we would have the Python code for the gaussian profile directly in the body of the `GaussianProfile` class. Does that work for you?

Reply:
Sounds good. I think it is good to do all of the above as part of the current PR, i.e. adding spatial chirp, and moving all of this to the `GaussianProfile` class.

Reply:
I've done these changes. There are some weird test and docs errors at the moment, but in any case I need to build a stand-alone test for the space-time cases. I will try to do it next week.